src/smart-components/myUserAccess/MUAContent.js

```diff
@@ -31,7 +34,7 @@ const MUAContent = ({ entitlements, isOrgAdmin }) => {
         </GridItem>
         <GridItem className="pf-m-9-col-on-md ins-l-myUserAccess-section__table">
           <Title headingLevel="h3" size="xl">
-            Your {isOrgAdmin ? 'roles' : 'permissions'}
+            {`${bundleData.find(({ entitlement }) => entitlement === bundle)?.title} ${isOrgAdmin ? 'roles' : 'permissions'}`}
```

Comment:
This should still have `Your` in it. 

`Your {bundle_name} roles` (or permissions)